qp_pacbio/scripts.py

```diff
@@ -111,3 +115,47 @@ def execute(url, job_id, output_dir):
 def finish_qp_pacbio(url, job_id, output_dir):
     """Executes the task given by job_id and puts the output in output_dir"""
     plugin(url, job_id, output_dir)
+
+
+@click.command()
+@click.option("--base", type=click.Path(exists=True), required=True)
+def biom_merge(base):
```

Comment:
This seems like a lot of logic to merge some BIOM. Is this borrowed from qp-woltka? Do you have tests supporting this function in particular?